pinc/Quiz.inc

```diff
@@ -168,28 +168,30 @@ class Quiz
             {
                 $passed = user_has_passed_quiz_page($username, $quiz_page_id);
                 $text = $passed ? _("Passed") : _("Not passed");
-                $bgcolor = $passed ? '#ccffcc' : '#ffcccc';
-                echo "<td style='background-color: $bgcolor'>$text</td>";
+                $bgcolor = $passed ? 'pass' : 'no-pass';
+                echo "<td class='$bgcolor'>$text</td>";
+
                 $date = get_last_attempt_date_for_quiz_page($username, $quiz_page_id);
                 $text = ($date != 0) ? strftime("%d-%b-%y", $date) : _("Not attempted");
                 $max = $this->pass_requirements['maximum_age'];
                 $date_ok = ((time() - $date) < $max) || empty($max);
-                $bgcolor = $date_ok ? '#ccffcc' : '#ffcccc';
-                echo "<td style='background-color: $bgcolor'>$text</td>";
-                $text = ($passed && $date_ok) ?	"<span style='font-size: 1.5em' title='Quiz page passed!'>&check;</span>"
-                    : "<span style='font-size: 1.5em' title='Quiz page not passed'>&#x2717;</span>";
-                $bgcolor = ($passed && $date_ok) ? '#88ff88' : '#ff8888';
-                echo "<td style='background-color: $bgcolor'>$text</td>";
+                $bgcolor = $date_ok ? 'date-ok' : 'date-not-ok';
```

Comment:
Similar as the above, consider prefixing with `quiz-`: `quiz-date-ok` & `quiz-date-not-ok`.